app/index.js

```diff
@@ -90,12 +90,13 @@ var AngularFullstackGenerator = yeoman.generators.Base.extend({
       this.prompt([{
           type: 'list',
           name: 'script',
+          default: 1,
           message: 'What would you like to write scripts with?',
           choices: [ 'JavaScript', 'CoffeeScript'],
           filter: function( val ) {
             var filterMap = {
-              'JavaScript': 'js',
-              'CoffeeScript': 'coffee'
+              'CoffeeScript': 'coffee',
+              'JavaScript': 'js'
```

Comment:
Is there a point to alphabetical sorting? I think it makes more sense to have JS first at least.